### ISSUE TITLE
Provide pre-built releases

### ISSUE THREAD

**Issue by RaymiiOrg** [lenticularis39/axpbox#22]
On the comp.os.vms mailing list [there was a request](https://groups.google.com/g/comp.os.vms/c/QFR_QeiUjoQ/m/bhO2UgmkBAAJ) to provide pre-built binaries. 

I'll see what I can do with github actions and releases (using that as well [for a personal project](https://github.com/RaymiiOrg/c_ookieclicker))

**Comment by lenticularis39** [comment]
Yeah, this would definitely be helpful for AXPbox users, especially for those who don't have a development environment installed.

About AXPbox on Windows: I'm not sure if it builds. I remember reading about someone trying to build es40 using Cygwin, but I suppose that was to handle the tricky "double configure" build system of es40. I can't think of anything that would prevent a CMake Windows build, but it's possible it fails because of some small defail... in any case if that happens a bug report should be created.

**Comment by RaymiiOrg** [comment]
I'll assign myself and when I have some spare time, get a linux build via github actions up as a demo.

**Comment by RaymiiOrg** [comment]
Alright, created merge request #26 - which included a github actions file. That is not the "releases" part, but does provide binaries, and can be used to create an actual release. If you like it I can continue on with the actual "releases" part.

**Comment by RaymiiOrg** [comment]
Did some more work on the windows msvc build, finally fixed the two linker errors, at least on github: https://github.com/RaymiiOrg/axpbox/tree/github_actions_build_windows_test

Will have to clean up the branch, rebase current and actually test on a windows system, but it now builds, which is a major step forward. 

If the windows build works, we have working automated builds for Linux, OS X and Windows, which means I can turn the "current" version into a release.

The other release, without all the recent changes, is a bit stalled. Cherry picking and rebasing is a bit more work  than I expected...

Not sure if we can provide Raspberry pi builds in an automated fashion, now that that seems to be a used architecture

**Comment by RaymiiOrg** [comment]
Seems to run okay on my Windows VM:

![image](https://user-images.githubusercontent.com/1927494/104117333-c428d000-5320-11eb-8cd6-021082a1b5b6.png)
